frontend/src/pages/Policy/policySections/DataCollectionTab.tsx

```diff
@@ -7,6 +12,194 @@ export default function DataCollectionTab() {
+								<p className='my-0 ml-2'>National- and state-level data</p>
+							</li>
+							<li className='flex flex-row align-center'>
+								<CheckRounded className='text-text text-altGreen' />
+
+								<p className='my-0 ml-2'>Breakdowns by race/ethnicity</p>
+							</li>
+							<li className='flex flex-row align-center'>
+								<BlockRounded className='text-text text-redOrange' />
+
+								<p className='my-0 ml-2'>Breakdowns by age</p>
+							</li>
+							<li className='flex flex-row align-center'>
+								<BlockRounded className='text-text text-redOrange' />
+
+								<p className='my-0 ml-2'>Breakdowns by sex</p>
+							</li>
+							<li className='flex flex-row align-center'>
+								<BlockRounded className='text-text text-redOrange' />
+
+								<p className='my-0 ml-2'>Breakdowns by cause of death</p>
+							</li>
+						</ul>
+					</div>
+
+					<div>
+						<p className='text-smallest'>
+							What’s included in our <HetTerm>general population</HetTerm>{' '}
+							dataset:
+						</p>
+						<ul className='list-none p-0 text-smallest'>
+							<li className='flex flex-row align-center'>
+								<CheckRounded className='text-text text-altGreen' />
+								<p className='my-0 ml-2'>National- and state-level data</p>
+							</li>
+							<li className='flex flex-row align-center'>
+								<CheckRounded className='text-text text-altGreen' />
+
+								<p className='my-0 ml-2'>Breakdowns by race/ethnicity</p>
+							</li>
+							<li className='flex flex-row align-center'>
+								<CheckRounded className='text-text text-altGreen' />
+
+								<p className='my-0 ml-2'>Breakdowns by age</p>
+							</li>
+							<li className='flex flex-row align-center'>
+								<CheckRounded className='text-text text-altGreen' />
+
+								<p className='my-0 ml-2'>Breakdowns by sex</p>
+							</li>
+							<li className='flex flex-row align-center'>
+								<CheckRounded className='text-text text-altGreen' />
+
+								<p className='my-0 ml-2'>Breakdowns by cause of death</p>
+							</li>
+						</ul>
+					</div>
+				</div>
+			</section>
+			<section id='#fatality-definitions'>
+				<h3 className='my-4 text-title font-medium text-altGreen'>
+					Fatality Definitions
+				</h3>
+				<StripedTable
+					id=''
+					applyThickBorder={false}
+					columns={[
+						{ header: 'Topic', accessor: 'topic' },
+						{
+							header: 'Measurement Definition',
+							accessor: 'measurementDefinition',
+						},
+					]}
+					rows={[
+						{
+							topic: 'Gun deaths (youth)',
+							measurementDefinition:
+								'Deaths of individuals under the age of 18 caused by firearms.',
+						},
+						{
+							topic: 'Gun homicides',
+							measurementDefinition:
+								'Deaths caused by firearms used with the intent to harm others.',
+						},
+						{
+							topic: 'Gun suicides',
+							measurementDefinition:
+								'Deaths resulting from individuals using firearms to inflict self-harm.',
+						},
+					]}
```

Comment:
We should split "youth" here into "children" and "young adult" since those are the sub data types, as are the homicides and suicides we list below